application/src/main/java/org/togetherjava/tjbot/commands/code/CodeMessageAutoDetection.java

```diff
@@ -73,27 +73,12 @@ public void onMessageReceived(MessageReceivedEvent event) {
         codeMessageHandler.addAndHandleCodeMessage(originalMessage, true);
     }
 
-    private boolean isSendByTopHelper(Member member) {
-        final Path configPath = Path.of("config.json");
-
-        try {
-            final Config config = Config.load(configPath);
-            final String[] tagManageRolePatterns = config.getTagManageRolePattern().split("\\|");
-
-            for (final String pattern : tagManageRolePatterns) {
-                for (final Role role : member.getRoles()) {
-                    if (role.getName().matches(pattern)) {
-                        return true;
-                    }
-                }
-            }
-
-        } catch (IOException e) {
-            logger.error("Unable to load the configuration file from path '{}'",
-                    configPath.toAbsolutePath(), e);
-        }
+    private boolean isCodeActionIgnored(Member member) {
+        List<Role> roles = member.getRoles();
```

Comment:
remove this method and use the value directly

Reply:
done.